R/pkgs/hospitalization/tests/testthat/test_hospitalization.R

```diff
@@ -1,9 +1,10 @@
 test_that("create_delay_frame works",{
+  require(data.table)
   X <- "input"
   varname <- "output"
   p_X <- 0.1
   X_pars <- c(10,1)
-  data_ <- data.frame(
```

Comment:
This is definitely not an appropriate change... I would rather count this test as failed than require the input to be data tables